src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -189,6 +190,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             switch (feature)
             {
                 // C# 7.3 features.
+                case MessageID.IDS_DefaultEquality: // semantic check
```

Comment:
>IDS_DefaultEquality [](start = 31, length = 19)

It looks like the convention is to have "Feature" as a prefix for feature IDs.
Also, why are we treating this is a feature? Can't we just view it as a bug fix?